Framework/PythonInterface/test/python/mantid/geometry/DetectorInfoTest.py

```diff
@@ -89,6 +90,13 @@ def test_twoTheta(self):
         info = self._ws.detectorInfo()
         self.assertEqual(type(info.twoTheta(0)), float)
 
+    def test_azimuthal(self):
+        """ See if the returned value is a double (float in Python). """
+        info = self._ws.detectorInfo()
+        self.assertEqual(type(info.azimuthal(0)), float)
+        x, y, _ = info.position(0)
+        self.assertEqual(np.arctan2(y, x), info.azimuthal(0))
```

Comment:
From a quick look at the code I think detector 0 is at x=y=0. Please verify. If that is the case this is not a sufficient test, I think.

Reply:
Andrei was asking me offline to improve this and check all of the detectors, not just 0.